shopfloor/tests/test_delivery_done.py

```diff
@@ -45,7 +45,7 @@ def test_done_all_qty_done(self):
         # Do not use the /set_qty_done_line endpoint to set done qties to not
         # update the picking to 'done' state automatically
         for move_line in self.picking.move_line_ids:
-            move_line.qty_done = move_line.quantity
+            move_line._pick_qty(move_line.quantity)
```

Comment:
Picking all reserved qty:
```suggestion
            move_line.picked = True
```